shared/constants/countryRegion.ts

```diff
@@ -1,256 +1,256 @@
-  Myanmar = 'MYANMAR',
-  Namibia = 'NAMIBIA',
-  Nauru = 'NAURU',
-  Nepal = 'NEPAL',
-  Netherlands = 'NETHERLANDS',
-  Netherlands_Antililles = 'NETHERLANDS ANTILILLES',
-  New_Caledonia = 'NEW CALEDONIA',
-  New_Zealand = 'NEW ZEALAND',
-  Nicaragua = 'NICARAGUA',
-  Niger = 'NIGER',
-  Nigeria = 'NIGERIA',
-  Niue_Island = 'NIUE ISLAND',
-  Norfolk_Island = 'NORFOLK ISLAND',
-  Norway = 'NORWAY',
-  Oman = 'OMAN',
-  Others = 'OTHERS',
-  Pacific_Island_Trust_Territory = 'PACIFIC ISLAND TRUST TERRITORY',
-  Pakistan = 'PAKISTAN',
-  Palau = 'PALAU',
-  Palestine = 'PALESTINE',
-  Panama = 'PANAMA',
-  Panama_Canal_Zone = 'PANAMA CANAL ZONE',
-  Papua_New_Guinea = 'PAPUA NEW GUINEA',
-  Paraguay = 'PARAGUAY',
-  Peru = 'PERU',
-  Philippines = 'PHILIPPINES',
-  Pitcairn_Island = 'PITCAIRN ISLAND',
-  Poland = 'POLAND',
-  Portugal = 'PORTUGAL',
-  Puerto_Rico = 'PUERTO RICO',
-  Qatar = 'QATAR',
-  Reunion = 'REUNION',
-  Romania = 'ROMANIA',
-  Russia = 'RUSSIA',
-  Rwanda = 'RWANDA',
-  Samoa = 'SAMOA',
-  San_Marino = 'SAN MARINO',
-  Sao_Tome_and_Principe = 'SAO TOME & PRINCIPE',
-  Saudi_Arabia = 'SAUDI ARABIA',
-  Senegal = 'SENEGAL',
-  Serbia = 'SERBIA',
-  Serbia_Montenegro = 'SERBIA/MONTENEGRO',
-  Seychelles = 'SEYCHELLES',
-  Sierra_Leone = 'SIERRA LEONE',
-  Singapore = 'SINGAPORE',
-  Slovak_Republic = 'SLOVAK REPUBLIC',
-  Slovenia = 'SLOVENIA',
-  Solomon_Islands = 'SOLOMON ISLANDS',
-  Somalia = 'SOMALIA',
-  South_Africa = 'SOUTH AFRICA',
-  Spain = 'SPAIN',
-  Sri_Lanka = 'SRI LANKA',
-  St_Helena = 'ST. HELENA',
-  St_Kitts_Nevis = 'ST. KITTS-NEVIS',
-  St_Lucia = 'ST. LUCIA',
-  St_Pierre_and_Miquelon = 'ST. PIERRE & MIQUELON',
-  St_Vincent = 'ST. VINCENT',
-  Sudan = 'SUDAN',
-  Suriname = 'SURINAME',
-  Swaziland = 'SWAZILAND',
-  Sweden = 'SWEDEN',
-  Switzerland = 'SWITZERLAND',
-  Syria = 'SYRIA',
-  Taiwan = 'TAIWAN',
-  Tajikistan = 'TAJIKISTAN',
-  Tanzania = 'TANZANIA',
-  Thailand = 'THAILAND',
-  Timor = 'TIMOR',
-  Togo = 'TOGO',
-  Tokelau_Islands = 'TOKELAU ISLANDS',
-  Tonga = 'TONGA',
-  Trinidad_and_Tobago = 'TRINIDAD & TOBAGO',
-  Tunisia = 'TUNISIA',
-  Turkey = 'TURKEY',
-  Turkmenistan = 'TURKMENISTAN',
-  Turks_and_Caicos_Islands = 'TURKS & CAICOS ISLANDS',
-  Tuvalu = 'TUVALU',
-  Uganda = 'UGANDA',
-  Ukraine = 'UKRAINE',
-  United_Arab_Emirates = 'UNITED ARAB EMIRATES',
-  United_Kingdom = 'UNITED KINGDOM',
-  United_States = 'UNITED STATES',
-  Unknown = 'UNKNOWN',
-  Upper_Volta = 'UPPER VOLTA',
-  Uruguay = 'URUGUAY',
-  US_Virgin_Islands = 'US VIRGIN ISLANDS',
-  Uzbekistan = 'UZBEKISTAN',
-  Vanuatu = 'VANUATU',
-  Vatican_City_State = 'VATICAN CITY STATE',
-  Venezuela = 'VENEZUELA',
-  Vietnam = 'VIETNAM',
-  Wake_Island = 'WAKE ISLAND',
-  Wallis_and_Futuna = 'WALLIS AND FUTUNA',
-  Western_Sahara = 'WESTERN SAHARA',
-  Yemen = 'YEMEN',
-  Yugoslavia = 'YUGOSLAVIA',
-  Zaire = 'ZAIRE',
-  Zambia = 'ZAMBIA',
-  Zimbabwe = 'ZIMBABWE',
+  Afghanistan = 'Afghanistan',
```

Comment:
should we leave the enum values in upper case (since it is shared and meant to be backwards compatible)? We can do the transformation on the client side just before.

it's also ok to leave it here like this. but note that validation for myinfo values might require processing in the future when they also use this type.

Reply:
Hmm, if I leave the enum values in upper case, that would mean I need to produce the correct capitalisation on render. This means I need another set of values as there is no known way to get to the right capitalisation with just JS built-in functions. Does not seem DRY to me.

Since we plan to render countries/regions in its appropriate capitalisation, I would recommend having the corrected values here and call an upper or lower-case function on validation.